src/data_snack/key_factories/non_cluster.py

```diff
@@ -5,7 +5,7 @@
 
 
 @dataclass
-class SingleKeyFactory(KeyFactory):
+class NoneClusterKeyFactory(KeyFactory):
```

Comment:
Shouldn't it be `NonCluster`... instead `None`?

Reply:
Fixed